app/scripts/modules/google/serverGroup/details/scalingPolicy/scalingPolicy.directive.html

```diff
@@ -0,0 +1,22 @@
+<dl ng-class="InsightFilterStateModel.filtersExpanded ? '' : 'dl-horizontal dl-wide'" style="margin-bottom: 20px">
+  <dt>
+    Target {{policy.basis}}
+    <help-field ng-if="policy.basisHelpKey" key="{{policy.basisHelpKey}}"></help-field>
```

Comment:
You shouldn't need the `ng-if` here - if the `key` is not found, the help bubble will not render.